src/hyperion/external_interaction/nexus/nexus_utils.py

```diff
@@ -63,8 +63,12 @@ def create_goniometer_axes(
     return Goniometer(gonio_axes, scan_points)
 
 
-def get_current_time():
-    return datetime.utcfromtimestamp(time.time()).strftime(r"%Y-%m-%dT%H:%M:%SZ")
+def get_start_and_predicted_end_time(time_expected: float) -> tuple[str, str]:
+    start = datetime.utcfromtimestamp(time.time())
+    end_est = start + timedelta(seconds=time_expected)
+    return start.strftime(r"%Y-%m-%dT%H:%M:%SZ"), end_est.strftime(
+        r"%Y-%m-%dT%H:%M:%SZ"
+    )
```

Comment:
Nit: Slightly cleaner I think, take it or leave it.
```suggestion
def get_start_and_predicted_end_time(time_expected: float) -> tuple[str, str]:
    time_format = r"%Y-%m-%dT%H:%M:%SZ"
    start = datetime.utcfromtimestamp(time.time())
    end_est = start + timedelta(seconds=time_expected)
    return start.strftime(time_format), end_est.strftime(time_format)
```